packages/babel-core/src/errors/rewrite-stack-trace.ts

```diff
@@ -63,11 +63,13 @@ function CallSite(filename: string): CallSite {
     isConstructor: () => false,
     isToplevel: () => true,
     getFileName: () => filename,
+    /* eslint-disable unicorn/no-useless-undefined */
     getLineNumber: () => undefined,
     getColumnNumber: () => undefined,
     getFunctionName: () => undefined,
     getMethodName: () => undefined,
     getTypeName: () => undefined,
+    /* eslint-enable unicorn/no-useless-undefined */
```

Comment:
You may want enable `checkArrowFunctionBody` https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-useless-undefined.md#checkarrowfunctionbody added in v50

Reply:
Amazing! Thanks!